graphql/src/main/java/com/scalar/db/graphql/GraphQlFactory.java

```diff
@@ -55,13 +56,22 @@ private GraphQLObjectType createQueryObjectType() {
   private GraphQLObjectType createMutationObjectType() {
     GraphQLObjectType.Builder builder = GraphQLObjectType.newObject().name("Mutation");
     for (TableGraphQlModel tableModel : tableModels) {
-      builder.field(tableModel.getMutationPutField());
-      builder.field(tableModel.getMutationDeleteField());
+      builder
+          .field(tableModel.getMutationPutField())
+          .field(tableModel.getMutationBulkPutField())
+          .field(tableModel.getMutationDeleteField())
+          .field(tableModel.getMutationBulkDeleteField())
+          .field(tableModel.getMutationMutateField());
     }
-    builder.field(
-        GraphQLFieldDefinition.newFieldDefinition().name("commit").type(Scalars.GraphQLBoolean));
-    builder.field(
-        GraphQLFieldDefinition.newFieldDefinition().name("abort").type(Scalars.GraphQLBoolean));
+    builder
+        .field(
+            GraphQLFieldDefinition.newFieldDefinition()
+                .name("commit")
```

Comment:
It's more like a question about the specification.
Although even with only queries, we need to commit/abort but Is it OK to have `commit` and `abort` in `Mutation`? 
Are they also included in `Query` as well?

Reply:
As we discussed on Friday, I'm moving the commit operation to the directive argument in #427. It will be possible to send commits with query operations as well.